src/main/index.ts

```diff
@@ -624,6 +625,20 @@ function setupTaskbarFeatures() {
         mode: isPlaying ? "normal" : "paused"
       });
     }
+
+    if (store.get("playback.autoClickSkipAd") && state.adPlaying) {
```

Comment:
This section is contained within the Setting up of Taskbar features 😅
I feel like this should be placed in a better location

Reply:
solved👍